osgood/base.py

```diff
@@ -37,7 +38,7 @@ def inner(*args: Any, **kwargs: Any) -> Any:
     return decorator
 
 
-def ozip(list_to_zip: list[Any], groups: int, pad: bool = False) -> list[tuple]:
+def ozip(list_to_zip: List[Any], groups: int, pad: bool = False) -> List[tuple]:
```

Comment:
newer version of Python prefers regular python `list` rather than `typing.List`

Reply:
doesn't work, will revisit

Reply:
What version of Python are you on. This is for version >=3.8 I believe

Reply:
3.8.5